dxr/utils.py

```diff
@@ -295,7 +295,16 @@ def split_content_lines(unicode):
     resulting lines.
 
     """
-    return unicode.splitlines(True)
+    lines = unicode.splitlines(True)
+    # Vertical Tabs and Form Feeds are treated as end-of-lines by splitlines
```

Comment:
Now, there are a whole scad of other chars that it also considers linebreaks: \x1c, \x1d, \x85, etc.: https://docs.python.org/2.7/library/stdtypes.html#unicode.splitlines. Do you think we ought to handle all of them while we're at it? The bother and non-future-proofness of that makes me wonder whether we'd be better off encoding to utf-8, calling str.splitlines, and then re-decoding. I wonder what's fastest.